### DISCUSSION TITLE
Re-direct in Elasticsuite Search

### DISCUSSION THREAD

**Discussion by productcontent-j** [Smile-SA/elasticsuite#2918] [Q&A] [answered]
Hello,
Is it possible put a notification first before a re-direct attached to a search term in the onsite search? 
I'm working with Elasticsuite Search + Magento. I've found out how to place a re-direct (without 'warning') to a search term via _Marketing --> Search term -> re-direct_, but I would like to place an notification first so the client itself could choose continue shopping on this  particular page or use the button that will lead to the re-directed page. 
It would be great if I could place this or within the autocomplete or in the results.
Hope anyone could help me out. Thanks!

**Comment by romainruaud** [comment, marked as answer]
Hi, this is not an Elasticsuite feature, Magento legacy can do this out-of-the-box even without Elasticsuite.

So in any case, that's not something we'll cover as part of the Elasticsuite roadmap.

Regards

**Reply by productcontent-j** [reply]
Ah, I understand. Thank you very much! Regards :)
